zeeguu/core/word_scheduling/basicSR/basicSR.py

```diff
@@ -42,7 +48,7 @@ def __init__(self, bookmark=None, bookmark_id=None):
 
     def update_schedule(self, db_session, correctness):
         if correctness:
-
+            # Why have CORRECTS_IN_A_ROW_FOR_LEARNED = 4 rather than 3?
```

Comment:
asked also below: do we still need corrects in a row with Leitner ?

Reply:
I will leave the code to keep track of the consecutive answers, but not use it in the logic and instead using the cooling interval. 

Should we update the strings in the Web explaining the "Learned" words? "Learned words are words that were correct in exercises on four different days or words that have been tagged to be 'Too easy'." Or is it OK still?

Reply:
Not necessarily. You could have a sequence like: C (1) -> C (2) -> C (3) -> W(2) -> C(3) -> C(4). I think we could just say "at least 4 different days" and that would fix the technicality.